Form-Controls/index.html

```diff
@@ -14,14 +14,38 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
+    <label for="userName">User Name</label>
+    <input type="text" id="userName" name="userName" minlength="2" maxlength="50" pattern="[a-zA-Z]{2,50}" title="Username should contain only letters (a-z, A-Z) and be 2 to 50 characters long." required placeholder="Enter User Name">
+    <label for="email">Email Address:</label>
+    <input type="email" id="email" name="email" required placeholder="Enter Email Address" pattern="[^@]+@[^@]+\.[a-zA-Z]{2,}"
```

Comment:
You are correct that it does nothing, however I attempted to prevent numbers but was unable. I believe it requires javascript to restrict the user from entering a number in the customer name. So thank you for your feedback.

Reply:
> You are correct that it does nothing, however I attempted to prevent numbers but was unable. I believe it requires javascript to restrict the user from entering a number in the customer name. So thank you for your feedback.

So, it's not in the acceptance criteria to prevent numbers. But it _is_ to only use HTML and CSS. So what's your strategy to meet the criteria? 

This isn't about criticising, to be clear,  it's about _detail_. Attention to detail is incredibly important in programming. We must be precise. Computers don't do vague. Setting up a methodical, precise way to go over the acceptance criteria, line by line, and checking you are meeting it, will really _really_ help you progress in programming. Try it!

Reply:
Thank you, Sally. I'm convinced that if I follow your guidance, I will be one of the most competitive developers. So, I'll do it. This is a unique input for me. Thank you, I am extremely pleased.